paserk.go

```diff
@@ -0,0 +1,209 @@
+package paseto
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// type of PASERK token
+type PaserkType int
+
+// Error indicating the given paserk import/export method is not yet implemented on the key type.
+type NotImplementedError struct {
+	keyTypeStr    string
+	paserkTypeStr string
+}
+
+// Error indicating the given paserk type is not valid for the key type.
+type InvalidPaserkTypeError struct {
+	keyTypeStr    string
+	paserkTypeStr string
+}
+
+const (
+	// Invalid Paserk token type
+	PaserkTypeInvalid PaserkType = 0
+	// Unique Identifier for a separate PASERK for local PASETOs
+	PaserkTypeLid PaserkType = 1
```

Comment:
Any reason not to use the strings corresponding to each variant here for simplicity?